src/sonic-ctrmgrd/ctrmgr/ctrmgrd.py

```diff
@@ -0,0 +1,622 @@
+#!/usr/bin/python3 -u
+
+import datetime
+import syslog
+import os
+import json
+import inspect
+import datetime
+from collections import defaultdict
+from swsscommon import swsscommon
+from sonic_py_common import device_info
+
+import kube_commands
```

Comment:
Organize imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports):

>Imports should be grouped in the following order:
>
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

Also organize alphabetically within each group.

Also, `datetime` is being imported twice.

Reply:
ok

Reply:
done